tests/ops/task/traversal_data.py

```diff
@@ -516,7 +517,7 @@ def combined_mongo_postgresql_graph(
     return mongo_dataset, postgres_dataset
 
 
-def manual_dataset(db_name: str, postgres_db_name) -> Dataset:
+def manual_dataset(db_name: str, postgres_db_name) -> GraphDataset:
```

Comment:
small nit. I _think_ this is right. Feel free to ignore if it's wrong
```suggestion
def manual_graph_dataset(db_name: str, postgres_db_name) -> GraphDataset:
```

Reply:
sure I can change the name of the test function!